pkg/profile/issuer/profile.go

```diff
@@ -45,7 +46,7 @@ type ProfileData struct {
 	OIDCClientParams            *OIDCClientParams `json:"oidcParams,omitempty"`
 	CredentialScopes            []string          `json:"credScopes,omitempty"`
 	LinkedWalletURL             string            `json:"linkedWallet,omitempty"`
-	// Todo #issue Add credential manifest issuer object
+	CredentialManifestIssuer    cm.Issuer         `json:"issuer,omitempty"`
```

Comment:
you do schema validation when `CredentialManifestIssuer` is supplied?  Better to fail here rather than failing during transaction later during WACI.

Add some go doc comments about usage and expected format of this field.

Reply:
From the spec, you'd need to make sure the ID is not empty:

> The object MUST contain a id property, and its value MUST be a valid URI string that identifies who the issuer of the credential(s) will be.

My CredentialManifest validation code makes sure that the ID isn't empty, but only if you unmarshal a full CredentialManifest... (which will happen during the transaction, causing a failure)

Reply:
#581 Validation of the manifest will be done as part of this todo. Which i will address in the follow up pr.